collector/collector.go

```diff
@@ -121,48 +138,62 @@ func NewPostgresCollector(logger log.Logger, dsns []string, filters ...string) (
 		}
 	}
 
+	p.Collectors = collectors
+
 	servers := make(map[string]*server)
 	for _, dsn := range dsns {
 		s, err := makeServer(dsn)
 		if err != nil {
 			return nil, err
 		}
+		// Manually provided servers are always classified as "primary"
+		s.isPrimary = true
+
+		// TODO(@sysadmind): We need to discover the downstream servers and add them here.
+		// if p.autoDiscoverDatabases {
+		// }
```

Comment:
I think we're intending to drop the `autoDiscoverDatabases` feature. IIRC It's misused by the queries.yaml to turn this exporter into a generic SQL exporter.

Reply:
I'm happy to hear that. I was going to ask you about the usage of this flag because it is challenging to understand from the code. I'm not surprised that it causes issues for users.

Reply:
Yea, it's a confusing mess. Part of why I haven't done much maintenance work on this exporter myself. :joy_cat: